INSTALL/libraries.sh

```diff
@@ -8,6 +8,7 @@ python3.6 -m pip install --user zstandard
 python3.6 -m pip install --user tqdm
 
 # Install tre regexp-up-to-edit-distance library
+sudo apt-get update
```

Comment:
Just to demonstrate how review comments on individual lines work:

This one is redundant & slightly wasteful  — if one runs ALL.sh, it will run python.sh (where I also added an `update`) before libraries.sh.  But I wanted it to be safe to run each script separately, and not depend on other scripts for correctness.

Reply:
:+1: